### ISSUE TITLE
Support QWen？

### ISSUE THREAD

**Issue by Vincent131499** [epfLLM/Megatron-LLM#96]
Great job!
[QWen](https://huggingface.co/Qwen/Qwen-14B) is an open-source model widely used by the community. Does it support the training of this model?

**Comment by martinjaggi** [comment]
good idea, it would be nice if someone could make a PR to support Qwen 1.5 or later Qwen 2. 

it should be quite easy as the architecture seems very similar to the ones we already support in the current code

**Comment by longxudou** [comment]
With megatron-llm, we train sailor/sailor2 model on qwen1.5/qwen2.5.  The codebase is released in [Megatron-Sailor2](https://github.com/sail-sg/Megatron-Sailor2/blob/main/run_qwen2.sh) :)
